dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/DefaultMergeService.java

```diff
@@ -67,12 +64,6 @@ public DefaultMergeService( SchemaService schemaService )
         {
             if ( schema.isIdentifiableObject() )
             {
-                if ( BaseIdentifiableObject_.SHARING.equals( property.getFieldName() )
-                    && SharingUtils.isUseLegacySharing( (BaseIdentifiableObject) target ) )
-                {
-                    continue;
-                }
-
```

Comment:
This was added long time ago as an attempt to fix the `mergeMode=MERGE` for new `Sharing` in `MetadataImportService`.
As the merge option is deprecated, this need to be removed.